packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -2499,6 +2499,31 @@ it('should change page size', async () => {
   expect(rows[49].querySelector('.iui-cell')?.textContent).toEqual('Name50');
 });
 
+it('should render number of rows selected for paginator', async () => {
+  const { container } = renderComponent({
+    data: mockedData(100),
+    pageSize: 10,
+    paginatorRenderer: (props) => <TablePaginator {...props} />,
+    isSelectable: true,
+  });
+
+  const rowCheckboxes = container.querySelectorAll(
+    '.iui-table-body .iui-row .iui-checkbox',
+  );
+
+  expect(container.querySelector('.iui-left span')).toBeNull();
+
+  fireEvent.click(rowCheckboxes[0]);
+  expect(container.querySelector('.iui-left span')?.textContent).toBe(
+    '1 row selected',
+  );
+
+  fireEvent.click(rowCheckboxes[1]);
+  expect(container.querySelector('.iui-left span')?.textContent).toBe(
+    '2 rows selected',
+  );
+});
```

Comment:
Maybe you could add a test case to test flatRows. So by selecting one row, ensure that the subrow count is also included in the paginator count.

Optionally, you could also test that the disbaled subrows count is not added to the total count.

Reply:
I think this test would be useful. Like testing what happens if you select a row with subrows. The row selection count should also include the subrow count.

Reply:
The test has now been updated and uses a table with subrows